tests/test_dagmc_file.py

```diff
@@ -44,7 +25,7 @@ def test_load_file_populate():
 def test_set_native_ranges():
     """Tests set_native_ranges
     """
-    single_cube = ds.DagmcStats(test_env[1]['input_file'])
+    single_cube = df.DagmcFile(test_env[1]['input_file'])
```

Comment:
discussion: I'm sure it was a prior review that resulted in this `test_env` data structure, but it seems a little odd now because you use a variable name that is strongly tied to the input file in question.

This might be a good topic for a software review meeting.... 

Some thoughts:
* should we just define the input file name in the test?
* should we use a more generic variable name, like `geometry_model` instead of `single_cube`?  Many tests might then actually work for multiple input files (which may or may not be valuable...)
* this structure creates a kind of magic number about which entry in `test_env` to use - should that be a dictionary?

Reply:
Something like this makes sense:
test_env = {'three_vols': 'tests/3vols.h5m', 'single_cube': 'tests/single-cube.h5m', 'pyramid': 'tests/pyramid.h5m'}